config/packages/messenger.yaml

```diff
@@ -0,0 +1,13 @@
+framework:
+    messenger:
+        transports:
+            # Uncomment the following line to enable a transport named "amqp"
+             amqp: '%env(MESSENGER_TRANSPORT_DSN)%'
```

Comment:
What if I don't want to send the queries to the transport?

Reply:
I'm going to look into that.

Reply:
By default, no message is sent to the transport. You need to specify which message goes to which transport.